src/main/java/com/rufus/bumblebee/controllers/ContainerController.java

```diff
@@ -21,21 +23,33 @@
 public class ContainerController {
 
     private final ContainerService service;
+    private final HistoryService<HistoryDto, String> historyService;
 
     @Autowired
-    public ContainerController(ContainerService service) {
+    public ContainerController(ContainerService service, HistoryService<HistoryDto, String> historyService) {
         this.service = service;
+        this.historyService = historyService;
     }
 
-    @PostMapping(path = "/add")
-    public ResponseEntity<ContainerDto> addContainer(@RequestBody ContainerRequest request) {
-        return new ResponseEntity<ContainerDto>(
-                service.createContainer(request.getName(), request.getAuth(), request.getReportType()), HttpStatus.OK
+    @PostMapping(path = "/add", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<ContainerDto> addContainer(@RequestBody ContainerRequest request) throws Exception {
+        return new ResponseEntity<>(
+                service.createContainer(request.getName(), request.getHistoryOn(), request.getReportType()), HttpStatus.OK
         );
     }
 
-    @DeleteMapping(path = "/remove/{cuid}")
-    public ResponseEntity<String> removeContainer(@PathVariable("cuid") @NotEmpty String cuid) throws NoResultException {
-        return new ResponseEntity<String>("The container with the ID " + service.removeContainer(cuid) + " was deleted", HttpStatus.OK);
+    @DeleteMapping(path = "/remove/{cuid}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> removeContainer(@PathVariable("cuid") @NotEmpty String cuid) throws Exception {
+        return new ResponseEntity<>("The container with the ID " + service.removeContainer(cuid) + " was deleted", HttpStatus.OK);
```

Comment:
такое сообщение больше подходит для логов.
Допустим ситуацию что кто -то будет с вашим сервисом общаться и вы будете отслылать ему такое полотно , то как ему завязывать свою логику  .. если response == 200 && message.equals ("loremipsum"). - это больше совет на будущее и не только к этому методу.
Я бы например ,создал enum ContainerResponse
и возвращал 

```suggestion
        return new ResponseEntity<>(ContainerResponse.CONTAINER_REMOVED,"The container with the ID " + service.removeContainer(cuid) + " was deleted", HttpStatus.OK);
```
Что -то типа такого , я думаю идея понятна.

Reply:
> такое сообщение больше подходит для логов.
> Допустим ситуацию что кто -то будет с вашим сервисом общаться и вы будете отслылать ему такое полотно , то как ему завязывать свою логику .. если response == 200 && message.equals ("loremipsum"). - это больше совет на будущее и не только к этому методу.
> Я бы например ,создал enum ContainerResponse
> и возвращал
> 
> Что -то типа такого , я думаю идея понятна.

Да, Вы правы . Тут лучше использовать Map<String,String> и тогда потенциальный клиент сможет это легко распарсить и использовать